package.json

```diff
@@ -66,7 +66,7 @@
     "route": "/acquisition-units",
     "okapiInterfaces": {
       "acquisitions-units": "1.1",
-      "users": "15.0 15.1"
+      "users": "15.1"
```

Comment:
please look at your release ticket and related interfaces https://issues.folio.org/browse/UIAC-19
also please check https://wiki.folio.org/display/REL/Release+process+in+Jira

Reply:
it looks like the ticket was closed by mistake, 15.1 should be fine https://github.com/folio-org/mod-users/blob/v17.0.0/descriptors/ModuleDescriptor-template.json#L7